database/sql/V079__auction_prices_token_index.sql

```diff
@@ -0,0 +1,2 @@
+-- Create index to improve `fetch_latest_token_price` query performance.
+CREATE INDEX auction_prices_token_auction_id_idx ON auction_prices (token, auction_id DESC);
```

Comment:
Leaving a reminder to myself that here is another performance reason why we will probably have to keep `auction_prices` table, and erase auction prices from `competition_auctions`. (there are a few more places in the code)